pkg/config/config_test.go

```diff
@@ -341,6 +372,21 @@ func TestComposeConfig(t *testing.T) {
 			Platform:         configv1.OpenStackPlatformType,
 			IsSingleReplica:  true,
```

Comment:
You've removed the `isSingleReplica` field above but don't seem to have updated the tests to have a new test for when it's a single replica. This is reducing the test coverage and should be fixed

Reply:
https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/77/files#diff-29cea1b5b831c8655c7155f43e2367cec73e66d1e338f8b3c7877a2f339b8811R359

It is set correctly